portal-impl/src/portal.properties

```diff
@@ -6209,6 +6209,16 @@
     #
     counter.increment=100
 
+    #
+    # You can further fine tune the counter increment for specific counter
+    # names. This entry will ensure that the counter name
+    # "auto.increment.object.field" or anything that starts with
+    # "auto.increment.object.field#" will only increment by 1.
+    #
+    # Env: LIFERAY_COUNTER_PERIOD_INCREMENT_PERIOD_AUTO_PERIOD_INCREMENT_PERIOD_OBJECT_PERIOD_FIELD
+    #
+    counter.increment.auto.increment.object.field=1
```

Comment:
Just curiosity: Where this is being used?

Reply:
@gabrielwas It's being used [here](https://github.com/liferay-objects/liferay-portal/blob/f1862d23211bf126b699f5f44c13271c60a6cdad/modules/apps/object/object-api/src/main/java/com/liferay/object/field/util/ObjectFieldUtil.java#L178-L180). 

The `CounterFinderImpl` is responsible for concatenating the `counter.increment.` prefix with the `auto.increment.object.field` suffix [here](https://github.com/carolmariaabb/liferay-portal/blob/a839674f12b37039198175bdd9a592cd531d1d0d/portal-impl/src/com/liferay/counter/service/persistence/impl/CounterFinderImpl.java#L270).

Reply:
Thx